### PR TITLE
Add option for acknowledgementBeep

### PR DESCRIPTION
**Pull request by itsmojo**

### CODE REVIEW

File: OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift [hunk 1]
```diff
@@ -12,6 +12,7 @@ public struct BasalScheduleExtraCommand : MessageBlock {
 
     public let blockType: MessageBlockType = .basalScheduleExtra
     
+    public let acknowledgeReminder: Bool
```
Comment (ps2) [inline OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift]: audibleConfirmation is probably a better name for this variable.

**Comment by ps2** [comment]
Looks like tests need to be updated with the new method signatures.

File: OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift (hunk 1 above)
Comment (itsmojo) [reply OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift]: I think that calling this audibleConfirmation would be confusing as this is just one of 3 different types of audible confirmations for the pod when programming insulin commands. The PDM calls the other two types of audio confirmations the Confidence Reminder and the Program Reminder. All of 3 of these different reminders (audio confirmations) are encoded into a single byte referred to in the wiki and in Loop app as the reminder byte. The PDM doesn’t apparently use nor have a name for this particular type of reminder (basically it's the first half of a Confidence Reminder which only beeps when the command with the $80 bit set in the Reminder byte is accepted), so I have been calling this bit an Acknowledge Reminder in the wiki and now in the Loop app to fit with the PDM naming scheme.

**Comment by itsmojo** [comment]
Updated the tests for the new signature methods

File: OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift (hunk 1 above)
Comment (ps2) [reply OmniKit/MessageTransport/MessageBlocks/BasalScheduleExtraCommand.swift]: It's just not a "reminder". I was confused about the purpose of this flag when I saw it, due to that. Perhaps audibleAcknowledge? audibleCommandReceiptConfirmation?

**Pull request closed without merging**